### PR TITLE
Assert problem

### PR DESCRIPTION
**Pull request by rllola**
see https://github.com/Zondax/filecoin-signing-tools/pull/218

### CODE REVIEW

File: signer-npm/js/src/utils.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ function bytesToAddress(payload, testnet) {
       }
       break;
     case ProtocolIndicator.ACTOR:
-      if (payload.slice(1).length !== 32) {
+      if (payload.slice(1).length !== 20) {
```
Comment (rllola) [inline signer-npm/js/src/utils.js:114]: Is it sure that actor protocol payload are 20 bytes ? https://filecoin-project.github.io/specs/#protocol-2-actor

The payload is a sha256 which if I not mistaken is 32 bytes ?

File: signer-npm/js/src/utils.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ function addressAsBytes(address) {
       payload = address_decoded.slice(0, -4);;
       checksum = Buffer.from(address_decoded.slice(-4));
 
-      if (payload.byteLength !== 32) {
+      if (payload.byteLength !== 20) {
```
Comment (rllola) [inline signer-npm/js/src/utils.js:74]: Same here... ?

File: signer-npm/js/src/utils.js (hunk 1 above)
Comment (jleni) [reply signer-npm/js/src/utils.js:114]: Yes, this does not seem correct...
Maybe @mrsmkl can provide more info?

**Comment by rllola** [comment]
In the Go and Rust lib it is 20 bytes and the spec are out of date...

see #243 

We should check if there is no other places where this is incorrect...

File: signer-npm/js/src/utils.js (hunk 1 above)
Comment (mrsmkl) [reply signer-npm/js/src/utils.js:114]: In the go implementation the actor address as computed similarly to secp256k1 address, https://github.com/filecoin-project/go-address/blob/master/address.go#L166

Also in your test cases, for example https://github.com/Zondax/filecoin-signing-tools/blob/dev/examples/manual_testvectors.json#L142
... that's why I changed this. Let's ask about whether the specs should be updated.

**Pull request merged**